vmaas_sync/vmaas_sync.py

```diff
@@ -275,19 +276,40 @@ def set_last_repobased_eval_tms(cur, timestamp: dt.datetime):
 
     @staticmethod
     def _vmaas_repos_modified_since(modified_since: str) -> list:
-        repos_json = {'repository_list': ['.*'], 'modified_since': modified_since}
-        r_json = utils.vmaas_post_request(VMAAS_REPOS_ENDPOINT, repos_json, requests.Session())
-        repos = [repo_name_key for repo_name_key in r_json['repository_list']]
+        """Get list of modified repose since `modified since`"""
+        page = 1
+        repos_list = [".*"]
+        repos_names = []
+        session = requests.Session()
+        while True:
+            repos_json = {"repository_list": repos_list, "page": page, "page_size": DEFAULT_PAGE_SIZE,
+                          "modified_since": modified_since}
+            r_json = utils.vmaas_post_request(VMAAS_REPOS_ENDPOINT, repos_json, session)
+            if r_json is None:
+                break
+            LOGGER.info('Updating list of repositories (page: %s, page_size: %s, pages: %s, modified_sice: %s)',
+                        page, r_json['page_size'], r_json['pages'], r_json['modified_since'])
+            repos = r_json['repository_list']
+            for repo in repos:
```

Comment:
You could iterate through `r_json['repository_list']` directly here, could you change it to `for repo in r_json['repository_list']` please?

Reply:
Sure